src/lib.rs

```diff
@@ -451,6 +452,73 @@ impl Model {
 
         Ok(Col(self.highs.num_cols()? - 1))
     }
+
+    /// Hot-starts at the initial guess. See HIGHS documentation for further details.
+    ///
+    /// # Panics
+    ///
+    /// If HIGHS returns an error status value.
+    ///
+    /// If the data passed in do not have the correct lengths.
+    /// `cols` and `col_duals` should have the lengths of `num_cols`.
+    /// `rows` and `row_duals` should have the lengths of `num_rows`.
+    pub fn set_solution(
+        &mut self,
+        cols: Option<&[f64]>,
+        rows: Option<&[f64]>,
+        col_duals: Option<&[f64]>,
+        row_duals: Option<&[f64]>,
+    ) {
+        self.try_set_solution(cols, rows, col_duals, row_duals)
+            .unwrap_or_else(|e| panic!("HiGHS error: {:?}", e))
+    }
+
+    /// Tries to hot-start using an initial guess by passing the column and row primal and dual solution values.
+    /// See highs_c_api.h for further details.
+    ///
+    /// # Panics
+    ///
+    /// If the data passed in do not have the correct lengths.
+    /// `cols` and `col_duals` should have the lengths of `num_cols`.
+    /// `rows` and `row_duals` should have the lengths of `num_rows`.
```

Comment:
Since this returns an error type, wouldn't it be better to return a clean error instead of panicking ?

Reply:
This is just a documentation error, isn't it?